### ISSUE TITLE
New feature: Resolve DNS names

### ISSUE THREAD

**Issue by sjurtf** [lord-kyron/terraform-provider-phpipam#64]
Under the Scan agent settings in phpipam, there are scans that can be enabled.

![Screenshot 2023-04-19 at 13 54 37](https://user-images.githubusercontent.com/2546167/233067600-6e9e8109-ac76-493d-a38f-3b2aa451b580.png)

This provider supports Host Check and Discover new host with these attributes:
``` 
include_in_ping        = true
host_discovery_enabled = true
  ```

My feature request would be to support the option of enabling resolving of DNS names.

**Comment by lord-kyron** [comment]
@pavel-z1 what do you think? Will this be possible to implement?

**Comment by pavel-z1** [comment]
Hi @sjurtf 
I checked phpIPAM documentation https://phpipam.net/#subnets
Seems phpIPAM don't have `resolveDNS` parameter for subnet API controller or this parameter not documented.

If you provide example of phpIPAM API usage which activate resolve DNS names option I guess we could add it.

**Comment by sjurtf** [comment]
Hi, 
you are right - it is missing from the documentation.

We are running phpipam 1.5.3 and the `resolveDNS` is present in the API and it is working as expected when I tested it now.

Working notes:
When I call /api/<appid>/subnets/<subnetid> the response includes a field named `resolveDNS` set to `"1"` which matched my expectations and the GUI setting.

I tried to modify it with PATCH, and I was able to confirm that it changed the Resolve DNS names setting in the GUI.

```
curl --request PATCH \
  --url https://<redacted>/api/<appid>/subnets/<subnetid> \
  --header 'Content-Type: application/json' \
  --header 'token: X' \
  --data '{"resolveDNS": "1"}'
```
